xarray/backends/api.py

```diff
@@ -318,6 +318,33 @@ def maybe_decode_store(store, lock=False):
     return maybe_decode_store(store)
 
 
+def open_rasterio(filename, add_latlon=False):
```

Comment:
Now that I've played around a little bit, I think keeping the default to `False` is the right approach